Doc/faq/programming.rst

```diff
@@ -419,7 +419,7 @@ What is the difference between arguments and parameters?
 
 :term:`Parameters <parameter>` are defined by the names that appear in a
 function definition, whereas :term:`arguments <argument>` are the values
-actually passed to a function when calling it.  Parameters define what types of
+actually passed to a function when calling it.  Parameters define what kind of
```

Comment:
Maybe we could make `kind of arguments` a link to https://docs.python.org/3/glossary.html#term-parameter as novice users may not know what it is exactly.